mobility_data/importers/charging_stations.py

```diff
@@ -31,16 +40,32 @@ def __init__(self, elem, srid=settings.DEFAULT_SRID):
         y = geometry.get("y",0) 
         self.point = Point(x, y, srid=srid)
         self.name = attributes.get("NAME", "")
-        self.address = attributes.get("ADDRESS", "")
-        temp_str = self.address.split(",")[1].strip()        
-        self.zip_code = temp_str.split(" ")[0]
-        self.city = temp_str.split(" ")[1]
+        address_field = attributes.get("ADDRESS", "").lstrip().split(",")
+        street_name, street_number = get_street_name_and_number(address_field[0])
+        # if len is >1 address contains zip and city
+        if (len(address_field)>1):
```

Comment:
Wrapping parenthesis around this conditional is redundant due to it being the only expression, this is related to operator precedence rules.

Lines 46, 48, 52.

Reply:
Fixed, too much Cg coding during weekend so the parentheses "mode" was still on.